osmosis-1/osmosis-1.assetlist.json

```diff
@@ -15376,6 +15376,47 @@
         "osmosis-main",
         "osmosis-unlisted"
       ]
+    },
+    {
```

Comment:
Please do not update the assetlist.json file directly. This is generated only.

Reply:
done